### PR TITLE
Ashton Api-Muncher baseline PR

### PR DESCRIPTION
**Pull request by ashtn**
[Herokuapp](http://fridgemuncher.herokuapp.com/)
[Trello Board](https://trello.com/b/FvWBIDAE)

# API Muncher
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| How did you go about exploring the Edamam API, how did you try querying the API? |  |
| Describe your API Wrapper.  How did you decide on the methods you created? |  |
| Describe an edge case or failure case test you wrote for your API Wrapper. |  |
| Explain how VCR aids in testing an API. |  |
| What is the Heroku URL of your deployed application? |  |
| Do you have any recommendations on how we could improve this project for the next cohort? |  |

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
```
Comment (CheezItMan) [inline .gitignore:9]: Need to add `.env` to your `.gitignore` file.  Can't spread those API keys!

**Review by CheezItMan — APPROVED** [review]
Looks good, just add `.env` to your gitignore!
